.gitlab-ci.yml

```diff
@@ -115,6 +159,9 @@ test_macos11_py27_pytorch:
     PYTHON: "2.7"
     TEST_PACKAGE: coremltools.converters.mil.frontend.torch
     WHEEL_PATH: build/dist/*cp27*10_16*
+  only:
+    changes:
+      - coremltools/converters/mil/frontend/torch/**/*.{py}
```

Comment:
We would like to run these tests whenever there is a change in `mil/mil` and `mil/backend` as well.

Reply:
Likewise for tensorflow, whenever there is a change in `mil` or `backend`, we would like to run converter tests as well.

Reply:
That’s what I’m think as well.. but then it’s hard to avoid any time consuming tests..

Reply:
Will update.